sycl/test/regression/swizzle-vec.cpp

```diff
@@ -0,0 +1,13 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+
+#include <CL/sycl.hpp>
+
+int main() {
+  cl::sycl::vec<int8_t, 2> inputVec = cl::sycl::vec<int8_t, 2>(0, 1);
```

Comment:
I suggest we extend existing test - e.g. https://github.com/intel/llvm/blob/sycl/sycl/test/basic_tests/vectors/vectors.cpp

Reply:
Transferred in new commit